### DISCUSSION TITLE
RSS Feed

### DISCUSSION THREAD

**Discussion by HassanMullah** [Kovah/LinkAce#611] [Support]
Can someone please show me how an RSS feed should present itself in LinkAce.

For example, when I go to https://myurl.mysampledomain.tld/guest/lists/1/feed, I am redirected to https://myurl.mysampledomain.tld/dashboard in Firefox. The RSS feed readers don't show anything either. I have the Simple Docker Compose installation with Traefik v.2 running.

Kind regards

H.

**Comment by Kovah** [comment]
This redirection happens because you are logged in. You should be able to see the feed in a private browser window.

You can test this in the demo. After logging in, try to open https://demo.linkace.org/guest/lists/2/feed
